js/app.js

```diff
@@ -0,0 +1,65 @@
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const books = document.querySelector('#books-list');
+const addBtn = document.getElementById('form');
+const rmvBtn = document.getElementById('remove-btn');
+let booksArray;
+
+function displayBooks() {
+  const data = localStorage.booksList;
+  if (data === null) {
+    booksArray = [];
+  } else {
+    booksArray = JSON.parse(data);
+  }
+
+  let htmlCode = '';
+  booksArray.forEach((item, index) => {
+    htmlCode += `<div>
+    <h2>${item.title}</h2>
+    <p>${item.author}</p>
+    <button onclick='removeBook(${index})' id='remove-btn'>remove</button>
+    <hr>
+    </div>`;
+  });
+  books.innerHTML = htmlCode;
+}
```

Comment:
- [ ] good job looping through the array 👍 please make sure to set up the set and get items for localStorage correctly.